backend/apps/volontulo/tests/models/test_factories.py

```diff
@@ -0,0 +1,77 @@
+"""
+test module for factories
+"""
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from apps.volontulo.factories.factories import UserFactory, OrganizationFactory
+from apps.volontulo.models import Organization
+
+
+class UserFactoryTestCase(TestCase):
+    """Test for UserFactory"""
+    def setUp(self):
+        u"""setting up each test"""
+        # create user
+        UserFactory.create(first_name="Kornik", last_name="WDrzewie")
+        # create fake user
+        UserFactory.create()
+        self.last_add_user = User.objects.last()
```

Comment:
I don't believe, that `last` will return last added object alway (it's not promised), so maybe intead of
```
UserFactory.create()
self.last_add_user = User.objects.last()
```
we can simply do something like that:
```
self.totally_fake_user = UserFactory.create()
```